src/logs.py

```diff
@@ -1,28 +1,42 @@
+# This code imports the `glob`, `os`, and `time` modules
 import glob
 import os
 import time
 
+# The `Logging` class is defined
 class Logging:
+    # The class constructor initializes the `logFileOpened` attribute as `False`
```

Comment:
^
A better comment would be to explain what that attribute does, but that's tenuous.